### DISCUSSION TITLE
Install homebrew on mac m1 freeze at receiving objects

### DISCUSSION THREAD

**Discussion by zhazi0101** [Homebrew/discussions#3930] [Getting started]
Hi guys, 

I am trying to install homebrew on the M1 mac using following command line.
`/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"`

However, the process always stuck at 4% of receiving object step as below:

```
......
==> /usr/bin/sudo /bin/chmod go-w /opt/homebrew/share/zsh /opt/homebrew/share/zsh/site-functions
==> /usr/bin/sudo /usr/sbin/chown wen.yan /opt/homebrew/bin /opt/homebrew/etc /opt/homebrew/include /opt/homebrew/lib /opt/homebrew/sbin /opt/homebrew/share /opt/homebrew/var /opt/homebrew/opt /opt/homebrew/share/zsh /opt/homebrew/share/zsh/site-functions /opt/homebrew/var/homebrew /opt/homebrew/var/homebrew/linked /opt/homebrew/Cellar /opt/homebrew/Caskroom /opt/homebrew/Frameworks
==> /usr/bin/sudo /usr/bin/chgrp admin /opt/homebrew/bin /opt/homebrew/etc /opt/homebrew/include /opt/homebrew/lib /opt/homebrew/sbin /opt/homebrew/share /opt/homebrew/var /opt/homebrew/opt /opt/homebrew/share/zsh /opt/homebrew/share/zsh/site-functions /opt/homebrew/var/homebrew /opt/homebrew/var/homebrew/linked /opt/homebrew/Cellar /opt/homebrew/Caskroom /opt/homebrew/Frameworks
==> /usr/bin/sudo /usr/sbin/chown -R wen.yan:admin /opt/homebrew
==> Downloading and installing Homebrew...
remote: Enumerating objects: 220323, done.
remote: Counting objects: 100% (47/47), done.
remote: Compressing objects: 100% (47/47), done.
Receiving objects:   4% (8813/220323), 3.78 MiB | 1.79 MiB/s
```

I did have command line tool xcode installed and I rebooted my machine a couple of times, but no luck so far. Please note that every time it stuck at this 4% of this "Receiving objects" step. I wait for about one hour and no progress at all. Please help.

**Comment by SMillerDev** [comment]
That sounds like #666

**Reply by zhazi0101** [reply]
Hey @SMillerDev , thanks for the feedback. However, I am not sure they are the same issue. This issue #666 has the ref error (in their log, you can see "error: Not a valid ref: refs/remotes/origin/master"). But on my side, there is no error at all. Actually, I believe my case passed that ref checking phase and already hit the "receiving objects" step.

**Comment by zhazi0101** [comment]
Found the problem. This is caused by MacAfee company installed. Once disabled it, homebrew was successfully installed.
